deploy/ephemeral.yaml

```diff
@@ -637,6 +643,14 @@ objects:
                         "type": "keyword"
                       }
                     }
+                  },
+                  "sap": {
+                    "type": "object",
+                    "properties": {
```

Comment:
Should all of the properties be added here? Including `sids`, `versions` and `instance`?